skyportal/handlers/api/followup_request.py

```diff
@@ -440,6 +440,11 @@ def get(self, followup_request_id=None):
         )
 
         total_matches = followup_requests.count()
+        if total_matches > 1000:
+            return self.error(
+                f'More than 1000 followup requests ({total_matches}) match your request... please make your query more restrictive and try again'
+            )
+
```

Comment:
What I meant is that the number per page can never exceed some high number (e.g., 1000). If a user inputs None or >1000 the API returns an error. So you could always send a large query but you won't be able to ask for very long pages.

Reply:
@guynir42 ah yeah. makes sense.